docs/zh/docs/advanced/custom-response.md

```diff
@@ -0,0 +1,212 @@
+# 自定义响应 - HTML，流，文件和其他
+
+**FastAPI** 默认会使用 `JSONResponse` 返回响应。
+
+你可以通过直接返回 `Response` 来重载它，参见 [直接返回响应](response-directly.md){.internal-link target=_blank}。
+
+但你可以直接返回 `Response`，返回数据不会自动转换，也不会自动生成文档（例如，在 HTTP 头 `Content-Type` 中包含特定的「媒体类型」作为生成的 OpenAPI 的一部分）。
+
+你还可以在 *路径操作装饰器* 中声明你想用的 `Response`。
+
+你从 *路径操作函数* 中返回的内容将被放在该 `Response` 中。
+
+并且如果该 `Response` 有一个 JSON 媒体类型（`application/json`），比如使用 `JSONResponse` 或者 `UJSONResponse` 的时候，你返回的数据会自动转换成任何你在 *路径操作装饰器* 中声明的 Pydantic 的 `response_model` 类型。
+
+!!! note "说明"
+    如果你使用不带有任何媒体类型的响应类，FastAPI 认为你的响应没有任何内容，所以不会在生成的OpenAPI文档中记录响应格式。
+
+## 使用 `ORJSONResponse`
+
+例如，如果你需要压榨性能，你可以安装并使用 <a href="https://github.com/ijl/orjson" class="external-link" target="_blank">`orjson`</a> 并将响应设置为 `ORJSONResponse`。
+
+导入你想要使用的 `Response` 类（子类）然后在 *路径操作装饰器* 中声明它。
+
+```Python hl_lines="2 7"
+{!../../../docs_src/custom_response/tutorial001b.py!}
+```
+
+!!! info "提示"
+    参数 `response_class` 也会用来定义响应的「媒体类型」。
+
+    在这个例子中，HTTP 头的 `Content-Type` 会被设置成 `application/json`。
+
+    并且在 OpenAPI 文档中也会这样记录。
+
+!!! tip "小贴士"
+    `ORJSONResponse` 目前只在 FastAPI 中可用，而在 Starlette 中不可用。
+
+
+
+## HTML 响应
+
+使用 `HTMLResponse` 来从 **FastAPI** 中直接返回一个 HTML 响应。
+
+* 导入 `HTMLResponse`。
+* 将 `HTMLResponse` 作为你的 *路径操作* 的 `content_type` 参数传入。
```

Comment:
There is a mistake in english docs,the `content_type` here should be "response_class"